### PR TITLE
Muki lab12 atm

### PR DESCRIPTION
**Pull request by Muki-has-ideas**
atm lab.  misslabeled as lab12 not lab 10

### CODE REVIEW

**Comment by Muki-has-ideas** [comment]
double checked and just did another push

**Comment by Muki-has-ideas** [comment]
Also named this wrong at the get go. this is really for lab 10
^_^

File: code/muki/lab12.py [hunk 1]
```diff
@@ -58,14 +62,14 @@ def print_transactions(self, list):
     command = input('Enter a command:\nbalance\ndeposit\nwithdraw\ninterest\ntransaction log\nhelp\nexit\n\n>')
     if command == 'balance':
         balance = atm.check_balance() # call the check_balance() method
-        print(f'\nYour balance is ${balance}\n')
+        print(f'\nYour balance is ${amount}\n')
     elif command == 'deposit':
-        amount = float(input('How much would you like to deposit? '))
```
Comment (violette-git) [inline code/muki/lab12.py]: Currently, if you run the code then ask for a balance, you will get an error that says that amount was not defined. consider making amount a variable that starts at 0. 

`amount = 0` 

Also, consider separating your user interactions from the functions themselves. Right now your formulas aren't working correctly. Start by giving amount a value and start from there. Let me know if you get stuck

**Review by violette-git — DISMISSED** [review]
I suggest testing your program a little more, step by step. Start by using a lot of print statements to see when and where values change.

**Review by brucestull — APPROVED** [review]
[Lab 10: ATM](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/10%20ATM.md)

**Completed**

Lab meets requirements. Nice functionality of displaying account transactions.

**Pull request merged**